docs/README.md

```diff
@@ -39,7 +39,7 @@ The `add-to-cart-button` is a block responsible for adding products in the [Mini
 
 | Prop name               | Type      | Description                                                                       | Default value        |
 | ----------------------- | --------- | --------------------------------------------------------------------------------- | -------------------- |
-| `onClickBehavior`       | `enum` | Controls what happens when users click the button. Possible values are: `go-to-product-page` or `add-to-cart`. | `add-to-cart`              |
+| `onClickBehavior`       | `enum` | Controls what happens when users click the button. Possible values are: `go-to-product-page`, `add-to-cart` and `ensure-sku-selection`, which means that users will be redirected to the product's page if there are multiple available SKUs, and the product would be added to the cart otherwise. | `add-to-cart`              |
```

Comment:
```suggestion
| `onClickBehavior`       | `enum` | Controls what happens when users click on the button. Possible values are: `go-to-product-page`, `add-to-cart`, and `ensure-SKU-selection` (if multiple SKUs are available, users will be redirected to the product page to select the desired one. If the product only has 1 SKU available, it will be added to the cart once the button is clicked on). | `add-to-cart`              |
```

Reply:
One tiny detail, should we use `SKU`, all caps? In that case, I need to update the implementation to take that into account.

Reply:
hahaha NOOOOO! My mistake here! Let me fix it.